Sources/FileSystem/Model/BetaTester.swift

```diff
@@ -2,12 +2,13 @@
 
 import Foundation
 
-public struct BetaTester: Codable, Hashable {
-    public var email: String
-    public var firstName: String
-    public var lastName: String
+struct BetaTester: Codable, Hashable {
 
-    public init(
+    var email: String
+    var firstName: String
+    var lastName: String
+
+    init(
```

Comment:
Did you also want to remove the init for BetaTester in Filesystem model as well? I saw you remove the init for BetaGroup

Reply:
The only reason I left it in was because it was doing some work for us:  `?? ""`